packages/core/src/credential/Credential.spec.ts

```diff
@@ -37,16 +36,11 @@ import * as Attestation from '../attestation'
 import * as Claim from '../claim'
 import * as CType from '../ctype'
 import * as Credential from './Credential'
+import { getCTypeHashFromId } from '../ctype'
```

Comment:
Could we use `CType.getCTypeHashFromId` in this file instead of the new import?

Reply:
oh I missed that one